backends/edk2-compat/edk2-svc-read.c

```diff
@@ -9,14 +9,13 @@
 #include <errno.h>
 #include <string.h>
 #include <stdlib.h>
+#include <argp.h>
```

Comment:
Many of my previous comments re generate apply here; I'll recheck this file (and the other subcommands) after we conclude on them.